uint8 state

# Axis State enumerations
uint8 UNDEFINED = 0x0
uint8 IDLE = 0x1
uint8 STARTUP_SEQUENCE = 0x2
uint8 FULL_CALIBRATION_SEQUENCE = 0x3
uint8 MOTOR_CALIBRATION = 0x4
uint8 ENCODER_INDEX_SEARCH = 0x6
uint8 ENCODER_OFFSET_CALIBRATION = 0x7
uint8 CLOSED_LOOP_CONTROL = 0x8
uint8 LOCKIN_SPIN = 0x9
uint8 ENCODER_DIR_FIND = 0xA
uint8 HOMING = 0xB
uint8 ENCODER_HALL_POLARITY_CALIBRATION = 0xC
uint8 ENCODER_HALL_PHASE_CALIBRATION = 0xD
uint8 ANTICOGGING_CALIBRATION = 0xE
